humanize-duration.js

```diff
@@ -1088,6 +1090,14 @@
   // parameters.
   function humanizer(passedOptions) {
     var result = function humanizer(ms, humanizerOptions) {
+      if (
+        humanizerOptions !== undefined &&
+        humanizerOptions.hasOwnProperty("language") &&
+        humanizerOptions.language === "ar" &&
+        !humanizerOptions.hasOwnProperty("delimiter")
+      )
+        humanizerOptions.delimiter = " و ";
```

Comment:
This should instead be a property on the `ar` language so that other languages can use it.